File: Python/tests/test_bart_ewmv_minusrho.py
import pytest

from hbayesdm.models import bart_ewmv_minusrho


def test_bart_ewmv_minusrho():
    _ = bart_ewmv_minusrho(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_bart_ewmv_minusrho.R
context("Test bart_ewmv_minusrho")
library(hBayesDM)

test_that("Test bart_ewmv_minusrho", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_ewmv_minusrho(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
